From: Simon McVittie <smcv@debian.org>
Date: Thu, 24 Nov 2022 12:09:12 +0000
Subject: testsuite: Try enabling a11y tests,
 other than those known to be unstable

At least some of the tests implemented via the accessibility-dump
executable are known to be unstable, but the tests based on separate
executables (tree-performance.c, etc.) have been reasonably consistently
passing on ci.debian.net for several years, so hopefully they are also
reliable enough for upstream CI and we don't need to mark them as flaky?

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 testsuite/a11y/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testsuite/a11y/meson.build b/testsuite/a11y/meson.build
index 85d0f5d..ea6348c 100644
--- a/testsuite/a11y/meson.build
+++ b/testsuite/a11y/meson.build
@@ -100,7 +100,7 @@ foreach t: a11y_tests
                'G_TEST_BUILDDIR=@0@'.format(meson.current_build_dir()),
                'GSETTINGS_SCHEMA_DIR=@0@'.format(gtk_schema_build_dir),
              ],
-        suite: ['a11y', 'flaky'])
+        suite: ['a11y'])
 endforeach
 
 installed_test_data = [
